mart/attack/perturber/batch.py

```diff
@@ -43,6 +43,13 @@ def __init__(
 
         self.perturbers = torch.nn.ModuleDict()
 
+    def parameter_groups(self):
+        """Return parameters along with optim parameters."""
+        params = []
+        for _, perturber in self.perturbers.items():
```

Comment:
Why not just `self.perturbers.values()`?

Reply:
Replaced.